File: WebApp/app/model/Order.js
Ext.define('WebApp.model.Order', {
    extend: 'Ext.data.Model',
    fields: [
		{name : 'OrderId'}, 
		{name : 'CustomerId'}, 
		{name : 'OrderDate'},
		{name : 'ShipmentDate'},
		{name : 'OrderNumber'},
		{name : 'Status'},
		{name : 'Customer', mapping: 'Customer'},
		{name : 'OrderElement', mapping: 'OrderElement'},
	],
	hasMany : {
        model : 'OrderElement',
        associationKey : 'orderelementss'
    },
	hasOne : {
        model : 'Customer',
        associationKey : 'customer'
    }
});

File: WebApp/Models/Order.cs
namespace WebApp.Models
{
    using Newtonsoft.Json;
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Order")]
    public partial class Order
    {
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public Order()
        {
            OrderElement = new HashSet<OrderElement>();
        }

        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public Guid OrderId { get; set; }

        public Guid CustomerId { get; set; }

        [Column(TypeName = "date")]
        public DateTime OrderDate { get; set; }

        [Column(TypeName = "date")]
        public DateTime? ShipmentDate { get; set; }

        public int? OrderNumber { get; set; }

        [Required]
        [StringLength(255)]
        public string Status { get; set; }

        public virtual Customer Customer { get; set; }

        [JsonIgnore]
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<OrderElement> OrderElement { get; set; }
    }
}
